### DISCUSSION TITLE
Whisper desktop app for real time transcription and translation with help of some free translation API

### DISCUSSION THREAD

**Discussion by Dadangdut33** [openai/whisper#834] [Show and tell]
Hello everyone, I would like to share my own take on making a desktop application using Whisper model. It is simple and customizable. Some of the code are inspired by the people here so I would like to thank everyone that share their projects 🙏 

What this application can do:
- Real time transcription/translation using `speaker` (for windows only) or `mic` as input.
- Batch import files and wait for it to process and got a result of `.srt` (with timestamp) and `.txt` of each file process. 

The `UI` is made using `tkinter` and the code that I did is mainly for Windows but I'm pretty sure it could work on other OS such as mac and Linux. 

If you are interested feel free to check it here: https://github.com/Dadangdut33/Speech-Translate/

<details open>
    <summary>Preview</summary>
    <p align="center">
      <img src="https://raw.githubusercontent.com/Dadangdut33/Speech-Translate/master/assets/5.png" alt="Detached window preview">
    </p>
  </details>

**Comment by ammarsufyan** [comment]
cool!
